manifests/init.pp

```diff
@@ -308,6 +318,10 @@
   $user                          = $puppet::params::user,
   $group                         = $puppet::params::group,
   $dir                           = $puppet::params::dir,
```

Comment:
Maybe we can use ```validate_absolute_path``` on all these dirs, though I'm not sure if that also works for windows paths.